contracts/root/depositManager/DepositManager.sol

```diff
@@ -49,6 +49,23 @@ contract DepositManager is DepositManagerStorage, IDepositManager, IERC721Receiv
     }
   }
 
+  function depositBulk(address[] calldata _tokens, uint256[] calldata _amountOrTokens, address _user)
+    external
+  {
+    require(
+      _tokens.length == _amountOrTokens.length,
+      "Invalid Input"
+    );
+    for (uint256 i = 0; i < _tokens.length; i++) {
+      // will revert if token is not mapped
+      if (registry.isTokenMappedAndIsErc721(_tokens[i])) {
+        depositERC721ForUser(_tokens[i], _user, _amountOrTokens[i]);
+      } else {
+        depositERC20ForUser(_tokens[i], _user, _amountOrTokens[i]);
```

Comment:
will/should never come here if require in fails in `isTokenMappedAndIsErc721`(will revert).

Reply:
Will not come here if reverts, but it will reach here if isTokenMappedAndIsErc721 returns false.